File: app/src/main/java/nl/rsdt/japp/jotial/maps/clustering/ClusterManagerInterface.java
package nl.rsdt.japp.jotial.maps.clustering;

import java.util.ArrayList;
import java.util.Collection;

import nl.rsdt.japp.jotial.data.structures.area348.ScoutingGroepInfo;

/**
 * Created by mattijn on 07/08/17.
 */

public interface ClusterManagerInterface {
    void addItems(ArrayList<ScoutingGroepInfo> buffer);

    void cluster();

    Collection<ScoutingGroepInfo> getItems();

    void clearItems();
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/clustering/ClusterManagerInterface.kt
package nl.rsdt.japp.jotial.maps.clustering

import nl.rsdt.japp.jotial.data.structures.area348.ScoutingGroepInfo
import java.util.*

/**
 * Created by mattijn on 07/08/17.
 */

interface ClusterManagerInterface {

    val items: Collection<ScoutingGroepInfo>
    fun addItems(buffer: ArrayList<ScoutingGroepInfo>)

    fun cluster()

    fun clearItems()
}
